src/Files.Uwp/ViewModels/ItemViewModel.cs

```diff
@@ -387,13 +389,15 @@ private async void UserSettingsService_OnSettingChangedEvent(object sender, Sett
             switch (e.SettingName)
             {
                 case nameof(UserSettingsService.PreferencesSettingsService.ShowFileExtensions):
+                case nameof(UserSettingsService.PreferencesSettingsService.ShowThumbnails):
```

Comment:
Since it's a privacy feature. I'm not sure if it needs to be logged. Need opinion from @yaichenbaum

Reply:
AppCenter doesn't track any personal data, so this is fine.